### PR TITLE
lxterminal: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/lxterminal.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# lxterminal
+
+> Terminal emulator for LXDE.
+> More information: <https://wiki.lxde.org/en/LXTerminal>.
+
+- Open a lxterminal window:
```
Comment (bl-ue) [inline pages/linux/lxterminal.md:6]:
```suggestion
- Open an lxterminal window:
```

File: pages/linux/lxterminal.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`lxterminal`
+
+- Open a lxterminal window, run a command and exit:
```
Comment (bl-ue) [inline pages/linux/lxterminal.md:10]:
```suggestion
- Open an lxterminal window, run a command and exit:
```

File: pages/linux/lxterminal.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`lxterminal -e "{{command}}"`
+
+- Open a lxterminal window with multiple tabs:
```
Comment (bl-ue) [inline pages/linux/lxterminal.md:14]:
```suggestion
- Open an lxterminal window with multiple tabs:
```

File: pages/linux/lxterminal.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`lxterminal --tabs={{tab_name1,tab_name2...}}`
+
+- Open a lxterminal window with a specific title:
```
Comment (bl-ue) [inline pages/linux/lxterminal.md:18]:
```suggestion
- Open an lxterminal window with a specific title:
```

File: pages/linux/lxterminal.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`lxterminal --title={{title_name}}`
+
+- Open a lxterminal window with a specific working directory:
```
Comment (bl-ue) [inline pages/linux/lxterminal.md:22]:
```suggestion
- Open an lxterminal window with a specific working directory:
```

**Review by bl-ue — COMMENTED** [review]
5 equivalent & minor comments.  Assuming you pronouce `lxterminal` as `L X terminal`, `an` not `a` is used.

File: pages/linux/lxterminal.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/linux/lxterminal.md:6]: I thought that as well however its not a vowel.

Comment (bl-ue) [reply pages/linux/lxterminal.md]: No, it isn't.  But you're supposed to use `an` before if either (1) the first letter of the next word is a vowel, or (2) the word starts with a phonetic vowel. `lxterminal` phonetically spelled out is `elexterminal`, which does begin with a vowel 😉

**Review by bl-ue — APPROVED** [review]

File: pages/linux/lxterminal.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# lxterminal
+
+> Terminal emulator for LXDE.
+> More information: <https://wiki.lxde.org/en/LXTerminal>.
+
+- Open an lxterminal window:
```
Comment (marchersimon) [inline pages/linux/lxterminal.md:6]:
```suggestion
- Open an LXTerminal window:
```
This seems to be the official name. Should we change it everywhere?

File: pages/linux/lxterminal.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+`lxterminal`
+
+- Open an lxterminal window, run a command and exit:
+
+`lxterminal -e "{{command}}"`
+
+- Open an lxterminal window with multiple tabs:
+
+`lxterminal --tabs={{tab_name1,tab_name2...}}`
```
Comment (marchersimon) [inline pages/linux/lxterminal.md:16]:
```suggestion
`lxterminal --tabs={{tab_name1,tab_name2,...}}`
```
I think that was what we agreed on.

File: pages/linux/lxterminal.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open an lxterminal window with a specific title:
+
+`lxterminal --title={{title_name}}`
+
+- Open an lxterminal window with a specific working directory:
+
+`lxterminal --working-directory={{/path/to/directory}}`
```
Comment (marchersimon) [inline pages/linux/lxterminal.md:24]: Relative paths are not possible?

Comment (CleanMachine1) [reply pages/linux/lxterminal.md:24]: It take a working directory, not a file

its just cd'ing into it

File: pages/linux/lxterminal.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`lxterminal --tabs={{tab_name1,tab_name2,...}}`
+
+- Open an lxterminal window with a specific title:
+
+`lxterminal --title={{title_name}}`
+
+- Open an lxterminal window with a specific working directory:
+
+`lxterminal --working-directory={{/path/to/directory}}`
```
Comment (CleanMachine1) [inline pages/linux/lxterminal.md:24]:
```suggestion
- Open an LXTerminal window, run a command and exit:

`lxterminal -e "{{command}}"`

- Open an LXTerminal window with multiple tabs:

`lxterminal --tabs={{tab_name1,tab_name2,...}}`

- Open an LXTerminal window with a specific title:

`lxterminal --title={{title_name}}`

- Open an LXTerminal window with a specific working directory:

`lxterminal --working-directory={{/path/to/directory}}`
```

File: pages/linux/lxterminal.md (hunk 8 above)
Comment (navarroaxel) [reply pages/linux/lxterminal.md:24]: @marchersimon is asking if this valid (I tested and it's valid)
```suggestion
`lxterminal --working-directory={{path/to/directory}}`
```

Comment (CleanMachine1) [reply pages/linux/lxterminal.md:24]: Where does it start from?

Comment (navarroaxel) [reply pages/linux/lxterminal.md:24]: from the current directory, e.g. you can use:

```bash
lxterminal --working-directory=..
```

Comment (CleanMachine1) [reply pages/linux/lxterminal.md:24]: .. from where

/home? from ~

Comment (navarroaxel) [reply pages/linux/lxterminal.md:24]: @CleanMachine1 from the current directory, yeah you can Use whatever you want:
`--working-directory=~/Downloads`, `--working-directory=../another_folder`, `--working-directory=/root`, etc.

**Comment by marchersimon** [comment]
@CleanMachine1 what about https://github.com/tldr-pages/tldr/pull/6042#discussion_r638414150?

**Comment by CleanMachine1** [comment]
Done

**Review by marchersimon — APPROVED** [review]

File: pages/linux/lxterminal.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+# lxterminal
+
+> Terminal emulator for LXDE.
+> More information: <https://wiki.lxde.org/en/LXTerminal>.
+
+- Open an LXTerminal window:
+
+`lxterminal`
+
+- Open an LXTerminal window, run a command and exit:
```
Comment (sbrl) [inline pages/linux/lxterminal.md:10]: Perhaps this will read better?
```suggestion
- Open an LXTerminal window, run a command, and then exit:
```

Comment (CleanMachine1) [reply pages/linux/lxterminal.md:10]: what about 
```suggestion
- Open an LXTerminal window, run a command then exit:
```

Comment (marchersimon) [reply pages/linux/lxterminal.md:10]: I think both `and exit` and `, and then exit` are fine, but `then exit` sounds a little strange.

Comment (bl-ue) [reply pages/linux/lxterminal.md:10]: Yes, it's a little hasty.  I like `, and then exit`.

**Pull request merged**